### PR TITLE
Fix indents and remove wrapper.

### PR DESCRIPTION
**Pull request by keithjjones**
Fixing a few things I saw.

### CODE REVIEW

**Review by bbannier — COMMENTED** [review]
While you are at it, would you mind fixing the mixed indents in the blocks starting at l.737 and l.742 as well?

**Comment by keithjjones** [comment]
Got it, thanks!

File: analyzer/protocol/ipsec/main.zeek [hunk 1]
```diff
@@ -672,8 +672,8 @@ function set_session(c: connection)
 event protocol_confirmation(c: connection, atype: Analyzer::Tag, aid: count) &priority=5
 	{
 	if ( atype == Analyzer::ANALYZER_SPICY_IPSEC_IKE_UDP ||
-	     atype == Analyzer::ANALYZER_SPICY_IPSEC_TCP ||
-	     atype == Analyzer::ANALYZER_SPICY_IPSEC_UDP )
+		 atype == Analyzer::ANALYZER_SPICY_IPSEC_TCP ||
```
Comment (bbannier) [inline analyzer/protocol/ipsec/main.zeek:675]: FWIW, I think this we'd actually like to keep: we indent this line with a tab like the `if` on the line above, and then do manual aligning with spaces. Indenting with tabs here will give inconsistent results depending on a user's tabwith.

**Review by bbannier — CHANGES_REQUESTED** [review]

File: analyzer/protocol/ipsec/main.zeek (hunk 1 above)
Comment (keithjjones) [reply analyzer/protocol/ipsec/main.zeek]: Switched to spaces.

**Pull request merged**